box64/box64.spec

```diff
@@ -0,0 +1,761 @@
+# Tests are disabled as they require x86_64 libraries to run
+%bcond tests 0
+
+%global forgeurl https://github.com/ptitSeb/box64
+
+%global common_description %{expand:
+Box64 lets you run x86_64 Linux programs (such as games) on non-x86_64 Linux
+systems, like ARM (host system needs to be 64-bit little-endian).}
+
+%define hash a9f4e72c4b4f15289c8eef0b9be0359e2acf50dc
+
+Name:           box64
+Version:        %{hash}
+Release:        %autorelease
+Summary:        Linux userspace x86_64 emulator with a twist, targeted at ARM64
+
+License:        MIT
+URL:            https://box86.org
+Source:         %{forgeurl}/archive/%{hash}.tar.gz
+
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  perl-podlators
+BuildRequires:  systemd-rpm-macros
+
+# box64 only supports these architectures
+ExclusiveArch:  aarch64 riscv64 ppc64le %{x86_64}
+
+Requires:       %{name}-data = %{version}-%{release}
+# These should not be pulled in on x86_64 as they can cause a loop and prevent
+# any binary from successfully executing (#2344770)
+%ifnarch %{x86_64}
+Recommends:     %{name}-binfmts = %{version}-%{release}
+%endif
+%ifarch aarch64
+Requires(post): %{_sbindir}/update-alternatives
+Requires(postun): %{_sbindir}/update-alternatives
+%endif
+
+%description    %{common_description}
+
+%package        data
+Summary:        Common files for %{name}
+BuildArch:      noarch
+
+%description    data %{common_description}
+
+This package provides common data files for box64.
+
+%ifnarch %{x86_64}
+%package        binfmts
+Summary:        binfmt_misc handler configurations for box64
+
+%description    binfmts %{common_description}
+
+This package provides binfmt_misc handler configurations to use box64 to
+execute x86_64 binaries.
+%endif
+
+%ifarch aarch64
+%package        adlink
```

Comment:
It took me almost two hours to build the Box64 RPM spec which built all of the packages by default. We should exclude the ones we don't use.

The ones we care about are (along with example devices):
- `asahai` = Apple Silicon
- `sd888` = Lenovo Thinkpad X13s
- `sd8g2` = AYN Odin 2
- `sdoryon` = Lenovo Thinkpad T14s Gen 6 (but this RPM is missing that, we need to use the `-D SDORYON1` build flag for it).